server/api/db/sqldb/db.py

```diff
@@ -2925,6 +2926,19 @@ def _update_feature_set_features(
                 )
                 update_labels(feature, labels)
                 feature_set.features.append(feature)
+            elif feature_name not in features_to_remove:
+                # get the existing feature from the feature set
+                feature = next(
```

Comment:
what happen when `feature` is None?

Reply:
if the feature is not in `features_to_remove` and not in `features_to_remove`, it is in the current features.
I will add a check just in case.